plugins/services/src/js/components/LogView.js

```diff
@@ -269,17 +269,9 @@ class LogView extends React.Component {
           {fullLog}
         </Highlight>
       </pre>,
-      <CSSTransitionGroup
-        key="log-go-down-button"
-        transitionAppear={true}
-        transitionName="button"
-        transitionAppearTimeout={350}
-        transitionEnterTimeout={350}
-        transitionLeaveTimeout={350}
-        component="div"
-      >
+      <TransitionGroup appear={true}>
```

Comment:
At some point this may have transitioned in, but there aren't currently any in/out transition styles to I could find for this one 🤷‍♂️ 

I'd recommend we just remove this TransitionGroup